test/suites/dev-tanssi-relay/ethereum-inbound-queue-v2/test_receive_tanssi_and_eth_token_from_eth.ts

```diff
@@ -0,0 +1,186 @@
+// @ts-nocheck
+
+import "@tanssi/api-augment";
+
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import { type ApiPromise, Keyring } from "@polkadot/api";
+import { encodeAddress, decodeAddress } from "@polkadot/util-crypto";
+import {
+    generateUpdate,
+    FOREIGN_ASSET_ID,
+    ETHEREUM_NETWORK_TESTNET,
+    generateOutboundMessageAcceptedLog,
+    SEPOLIA_SOVEREIGN_ACCOUNT_ADDRESS,
+    ETHEREUM_NETWORK_MAINNET,
+} from "utils";
+import type { KeyringPair } from "@moonwall/util";
+
+describeSuite({
+    id: "ETHINBV2TANSETH",
+    title: "Receive Tanssi and ETH Native Token from Ethereum",
+    foundationMethods: "dev",
+
+    testCases: ({ it, context }) => {
+        let polkadotJs: ApiPromise;
+        let alice: KeyringPair;
+        let isStarlight: boolean;
+        let ethNetworkId: number;
+
+        beforeAll(async () => {
+            polkadotJs = context.polkadotJs();
+            const keyring = new Keyring({ type: "sr25519" });
+            alice = keyring.addFromUri("//Alice", { name: "Alice default" });
+
+            const runtimeName = polkadotJs.runtimeVersion.specName.toString();
+            isStarlight = runtimeName === "starlight";
+
+            ethNetworkId = isStarlight ? ETHEREUM_NETWORK_MAINNET : ETHEREUM_NETWORK_TESTNET;
+
+            if (isStarlight) {
```

Comment:
instead of doing this, I think it's better to add IS_SNOWBRIDGE_V2 enabled, the runtime spec name, and compare it. like I do with outbound.